third_party/bigframes_vendored/pandas/io/gbq.py

```diff
@@ -107,11 +117,18 @@ def read_gbq(
                 `project.dataset.tablename` or `dataset.tablename`.
                 Can also take wildcard table name, such as `project.dataset.table_prefix*`.
                 In tha case, will read all the matched table as one DataFrame.
-            index_col (Iterable[str] or str):
+            index_col (Iterable[str], str, bigframes.enums.IndexKind):
                 Name of result column(s) to use for index in results DataFrame.
 
+                If an empty iterable, such as ``()``, a default index is
+                generated. Do not depend on specific index values in this case.
+
                 **New in bigframes version 1.3.0**: If ``index_cols`` is not
                 set, the primary key(s) of the table are used as the index.
+
+                **New in bigframes version 1.4.0**: Support
+                :class:`bigframes.enums.TypeKind` to override default index
+                behavior.
```

Comment:
TypeKind or DefaultIndexKind?

Reply:
DefaultIndexKind. Good catch. I had tried a few names for this but the refactor option didn't catch docstrings I think.